package.json

```diff
@@ -14,12 +13,10 @@
   },
   "devDependencies": {
     "babel-core": "^6.26.0",
-    "babel-eslint": "^10.0.1",
+    "babel-eslint": "9.0.0",
     "babel-jest": "^23.4.0",
```

Comment:
Are there any jest files left? I suspect there's more deps we can remove.

Reply:
There are tonnes of jest files :) Lots of unit tests and snapshots for components (which are great IMHO).